samples/algorithms/simple-grover/SimpleGrover.qs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Quantum.Samples.SimpleGrover {
     /// # Summary
```

Comment:
As discussed offline, I agree with deleting Host.cs. That said, it looks like Host.cs is being used on this docs page, such that we should adapt the Grover quickstart in coordination with doing so: https://github.com/MicrosoftDocs/quantum-docs-pr/blob/master/articles/quickstarts/search.md.

+@KittyYeungQ for awareness and coordination.

Reply:
Ah, sorry. I will leave the change in then, and leave it up to you to coordinate?

Reply:
That might work in the meantime, yes. I definitely agree with removing Host.cs, just want to make sure that the docs are not impacted as we do so. Thanks for the help!

Reply:
@cgranade Do you want to set up a suitable feature branch to retarget this to?

Reply:
Sounds good, I've gone on and done that, then. Thanks!